project/src/test/test-player-save-upgrader.gd

```diff
@@ -46,20 +46,15 @@ func test_163e_lost_erases_success() -> void:
 	assert_eq(PlayerData.level_history.results("rank/7d")[0].success, false)
 
 
-func test_1682_chat_history_preserved() -> void:
-	load_legacy_player_data("turbofat-1682.json")
-	
-	assert_eq(PlayerData.chat_history.chat_age("creature/boatricia/my_maid_died"), 5)
-
-
+## Note: There is no test for save data version 1682. The updater performs some chat history renames, but these chat
+## items are nonexistent in the newest version of the game because of the removal of free roam mode
```

Comment:
Move this up and change it from two pound signs to one. It's misleading to have it attached to `test_199c`

Reply:
Moved